src/main/scala/org/broadinstitute/dsde/agora/server/webservice/util/DockerHubClient.scala

```diff
@@ -52,5 +40,13 @@ object DockerHubClient {
     url
   }
 
+  private def unmarshal(response: HttpResponse): Future[List[DockerTagInfo]] = {
```

Comment:
It's annoying that you can't use some combination of `DockerTagInfoFormat` and the built-in marshaller for `List`, but I couldn't figure it out with a brief investigation